_posts/2026-1-17-op4a05.md

```
@@ -0,0 +1,147 @@
+## Mixed Quantification
+
+The purpose of hybrid quantization is to use data types with higher precision for specific layers in the specified model and lower data types for other layers to ensure the accuracy of the final result. If the accuracy of a quantized model cannot meet the requirements and cannot be improved by co-quantization (such as KLD quantization), then hybrid quantization can be used to avoid loss of accuracy.
+
+## Mixed Quantification Examples
+
+Taking the `uint8` quantization of `MobileNetV2_ImageNet` in the previous section as an example, this model directly uses ACUITY to quantize `uint8` and has a significant loss of accuracy compared to the floating-point model.
+
+**MobileNetV2_ImageNet** — compare the results of the floating-point model after direct quantization by uint8:
+
+**Float model inference results**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)